### PR TITLE
Add badges to the top of README.md

### PR DESCRIPTION
**Pull request by QuantumNovice**
The following badges were added
* Docs
* Python Version
* Contributions
* Hits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,10 @@
 [![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.me/TheAlgorithms/100) &nbsp;
 [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/TheAlgorithms) &nbsp;
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/TheAlgorithms/Python)
-
+[![HitCount](http://hits.dwyl.com/QuantumNovice/TheAlgorithms/Python.svg?style=flat)](http://hits.dwyl.com/TheAlgorithms/Python)
```
Comment (cclauss) [inline README.md]: Points to the wrong repo.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,10 @@
...
+[![Inline docs](http://inch-ci.org/github/TheAlgorithms/Python?branch=master)](http://inch-ci.org/github/TheAlgorithms/Python)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/TheAlgorithms/Python/issues)
+[![Python 3.7](https://img.shields.io/badge/python-3.7-blue.svg)](https://www.python.org/downloads/release/python-370/)
```
Comment (cclauss) [inline README.md]: Let's make this __Tested on Python 3.7__ because I do not think that we require our users to be on 3.7 vs. 3.6.

Comment (cclauss) [reply README.md]: Please fix the title of this PR to something more descriptive like: __Add badges to the top of README.md__

Should we add a badge that tells the reader if our Travis CI tests are passing?

File: README.md (hunk 1 above)
Comment (QuantumNovice) [reply README.md]: Corrected.

File: README.md (hunk 2 above)
Comment (QuantumNovice) [reply README.md]: I added badge for Travis-CI.
Also should we be using Circle CI along with travis-CI?

**Comment by cclauss** [comment]
Clicking __LGTM__ should link to the LGTM report.

**Comment by cclauss** [comment]
The first word on each badge should be capitalized.

"Gitter" is a brand name so it should be capitalized.

We should use or not use __on__ consistently: Chat "on" Gitter, Tested "on" Python 3.7.

Instead of "Build", can we use "Travis CI" because we currently run both Travis and LGTM?

**Comment by QuantumNovice** [comment]
> 
> 
> The first word on each badge should be capitalized.
> 
> "Gitter" is a brand name so it should be capitalized.
> 
> We should use or not use **on** consistently: Chat "on" Gitter, Tested "on" Python 3.7.
> 
> Instead of "Build", can we use "Travis CI" because we currently run both Travis and LGTM?

On it. Consistency is important.

**Comment by cclauss** [comment]
Extra credit: ?logo=LGTM ?logo=travis ?logo=pytho ?logo=gitter

**Comment by QuantumNovice** [comment]
> 
> 
>     * [ ]  Use or don't use non-blanking space (`&nbsp;`) _consistently_ to put whitespace between badges
> 
>     * [ ]  Remove **on** from Gitter
> 
>     * [ ]  Limit icons to _only one line_ when viewing the README -- Pick the ones that you think are _most_ important -- We want readers to  read the doc, not spend too much time parsing all the badges.

I kept the repo size. It's important in deciding whether to fork the repo locally or not.

**Comment by cclauss** [comment]
Putting a `space`before the `&nbsp;`add two spaces.  Be consistent in spacing.

**Comment by QuantumNovice** [comment]
> 
> 
> Putting a `space`before the `&nbsp;`add two spaces. Be consistent in spacing.



> 
> 
> Putting a `space`before the `&nbsp;`add two spaces. Be consistent in spacing.

Consistent spaces with `&nbsp` added.

**Comment by cclauss** [comment]
OK... My last comment is a real nit...  Too much __green.svg__ ;-)

**Comment by QuantumNovice** [comment]
> 
> 
> OK... My last comment is a real nit... Too much **green.svg** ;-)

I noticed it but didn't change. Here we go improving on that.

**Comment by cclauss** [comment]
Put PayPal back to green because money is green and then are we DONE?!?

**Comment by QuantumNovice** [comment]
Done. Sorry for inconvenience.

**Comment by cclauss** [comment]
No inconvenience...  It is good to get the small things right.

**Comment by cclauss** [comment]
LGTM alerts should point to https://lgtm.com/projects/g/TheAlgorithms/Python/alerts

**Pull request merged**